scripts/start-ganache

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
```

Comment:
I think we should do the `cd` in a different command, especially as right now this script would actually be going into `scripts/` before running the rest.

We should do something like:

```bash
BASEPATH="$( dirname "$(dirname "$0")" )"
mnemonic=$(node -p "require(require('path').join(\"${BASEPATH}\", 'src/ganache-vars')).MNEMONIC")

cd "$BASEPATH"

...
```

(We want `aragon-ganache/` to be at the top level of the repo).